test/Dialect/SV/generates.mlir

```diff
@@ -0,0 +1,93 @@
+// RUN: circt-opt %s | FileCheck %s
+// RUN: circt-opt %s | circt-opt | FileCheck %s
+// RUN: circt-opt %s --export-verilog -o %t.mlir | FileCheck %s --check-prefix=SV
+
+hw.module @PrintPath<> () -> () {
+  %fd = hw.constant 0x80000002 : i32
+  sv.initial {
+    sv.fwrite %fd, "%m\n"
+  }
+}
+
+hw.module @Case1<NUM : i8> () -> () {
+  %fd = hw.constant 0x80000002 : i32
+  sv.generate "foo_case": {
+    sv.generate.case #hw.param.decl.ref<"NUM"> [
+      case (0, "case0") {
+        sv.initial {
+          sv.fwrite %fd, "case 0\n"
+        }
+      }
+      case (1 : i64, "case1") {
+        hw.instance "print1" @PrintPath() -> ()
+      }
+      default ("dflt") {
```

Comment:
my comments about default are essentially, should this (at least internally, ignoring pretty printing) be:
`case (defaultCaseAttr, "dflt")`

Reply:
Or even a single case pattern with a null attr.